STEPS.md

```diff
@@ -0,0 +1,195 @@
+# How to build Awesome Tasks, step by step
+
+This document describes how to build Awesome Tasks step by step.
+We will be introducing new concepts on each step, so make sure you understand what's going on before going further! If you have any doubts, please ask your coach for help!
+
+## Step 1
+### From basic HTML markup and styling to a dynamic list of tasks
+
+We will start with a new HTML file called `index.html`. We will use it as the skeleton of our little app. Here is how it will look like:
+
+*1.1 —`index.html` base markup*
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
```

Comment:
I know this is **very** important, but if the browser defaults to a sensible value, we may leave this concept hidden.